classes/log/SubmissionFileEventLogEntry.inc.php

```diff
@@ -21,7 +21,7 @@
 define('SUBMISSION_LOG_FILE_UPLOAD',	0x50000001);
 define('SUBMISSION_LOG_FILE_DELETE',	0x50000002);
 define('SUBMISSION_LOG_FILE_REVISION_UPLOAD',	0x50000008);
-define('SUBMISSION_LOG_FILE_REVISION_DELETE',	0x50000009);
+define('SUBMISSION_LOG_FILE_EDIT',	0x50000009);
```

Comment:
Is this change in meaning for older data accommodated during upgrade?

Reply:
Oh, I think it's actually a copypasta error where I forgot to change the value for the new constant. I've restored `SUBMISSION_LOG_FILE_REVISION_DELETE` to preserve the record for old events.

I'll leave these as-is for now. But thinking ahead to work in 3.4/3.5 on the notifications, what do you think about recasting old log entries during an upgrade? So a migration might change all `SUBMISSION_LOG_FILE_REVISION_DELETE` logs to `SUBMISSION_LOG_FILE_DELETE` logs. Would that be a sensible way to handle this or is it better for us to hold onto the original record in its original meaning?

Reply:
Hmm, I'm not dead-set against revisiting old log entries, but it's nice to have a log that you can read without needing to refer back through the upgrade process if some kind of audit is needed.